### DISCUSSION TITLE
6/29 첫 모임!

### DISCUSSION THREAD

**Discussion by HangryDev** [Pseudo-Lab/Anxious-Lookout#12] [General]
본문에 후기가 작성됩니다!

**Comment by hidden719** [comment]
함경표 :
1.  graph user graph
2. 그래프 관련 usecase들 및 연구(뉴스레터)

LLM Agent 관련 논문 혹은 블로그 아시는 분 있으시면 추천부탁드립니다 !!

**Comment by kuongee** [comment]
민지수:
	1. Lil’Log (https://lilianweng.github.io/)
	2. AI 전반적인 내용들
